### PR TITLE
[9.0][NEW][WIP] New module account_invoice_group_picking

### PR DESCRIPTION
**Pull request by carlosdauden**
This module allows print invoices with picking info

@Tecnativa

### CODE REVIEW

**Review by alexcbr2 — APPROVED** [review]
Functionally tested in runbot

File: account_invoice_group_picking/models/sale_stock.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                vals = line._prepare_invoice_line(qty=qty)
+                vals.update({'invoice_id': invoice_id,
+                             'sale_line_ids': [(6, 0, [line.id])]})
+                self.env['account.invoice.line'].create(vals)
+
+
+class SaleOrderLineStockMove(models.Model):
+    _name = 'sale.order.line.stock.move'
+
+    sale_line_id = fields.Many2one(
```
Comment (pedrobaeza) [inline account_invoice_group_picking/models/sale_stock.py:53]: This table seems a bit strange. You should put fields in the proper models (account.invoice.line or sale.order.line) I think.

File: account_invoice_group_picking/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+from . import test_product_pricelist_direct_print
```
Comment (pedrobaeza) [inline account_invoice_group_picking/tests/__init__.py]: This import is not correct

File: account_invoice_group_picking/models/account_invoice.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+from openerp import _, api, fields, models, tools
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.model
+    @tools.ormcache('picking', 'order')
+    def _prepare_picking_info(self, picking, order, inv_line):
```
Comment (pedrobaeza) [inline account_invoice_group_picking/models/account_invoice.py:13]: Why do you pass `inv_line` if it's not used?

File: account_invoice_group_picking/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Invoice Group Picking",
+    "summary": "Print invoice lines grouped by picking",
```
Comment (pedrobaeza) [inline account_invoice_group_picking/__openerp__.py:7]: It's not only about printing

File: account_invoice_group_picking/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "9.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "http://www.tecnativa.com",
```
Comment (pedrobaeza) [inline account_invoice_group_picking/__openerp__.py]: https

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
There's no README file

**Comment by pedrobaeza** [comment]
There's no README file

**Comment by rafaelbn** [comment]
@pedrobaeza this is ready, could you make last review please?, thanks!

**Review by rafaelbn — APPROVED** [review]
Working with this module in production during more than 1 month.

**Comment by pedrobaeza** [comment]
There are comments unattended

**Comment by pedrobaeza** [comment]
Travis is red in this one. @carlosdauden can you please check?

File: account_invoice_group_picking/models/sale_stock.py [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+    move_invoice_link_ids = fields.One2many(
+        comodel_name='sale.order.line.stock.move',
+        inverse_name='sale_line_id',
+    )
+
+    @api.multi
+    def invoice_line_create(self, invoice_id, qty):
+        """
+        Overwrite original function to create one invoice line for each move.
```
Comment (pedrobaeza) [inline account_invoice_group_picking/models/sale_stock.py:20]: We must never overwrite functions, so a new way should be found.

**Comment by rafaelbn** [comment]
Please @carlosdauden could you check travis?
```

2017-12-13 08:13:04,567 5240 INFO openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: test_account_invoice_group_picking (openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking.TestAccountInvoiceGroupPicking)
2017-12-13 08:13:08,704 5240 INFO openerp_test openerp.addons.base.ir.ir_qweb: 'o.partner_id.lang' is not a valid language code, is an empty field or is not installed, falling back to en_US
2017-12-13 08:13:08,818 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: ERROR
2017-12-13 08:13:08,827 5240 INFO openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: ======================================================================
2017-12-13 08:13:08,827 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: ERROR: test_account_invoice_group_picking (openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking.TestAccountInvoiceGroupPicking)
2017-12-13 08:13:08,827 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: Traceback (most recent call last):
2017-12-13 08:13:08,827 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/build/OCA/account-invoicing/account_invoice_group_picking/tests/test_account_invoice_group_picking.py", line 70, in test_account_invoice_group_picking
2017-12-13 08:13:08,827 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     self.env['report'].get_html(invoice, 'account.report_invoice')
2017-12-13 08:13:08,828 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/api.py", line 248, in wrapper
2017-12-13 08:13:08,828 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return new_api(self, *args, **kwargs)
2017-12-13 08:13:08,828 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/addons/report/models/report.py", line 152, in get_html
2017-12-13 08:13:08,828 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     report_name, data=data, context=self._context)
2017-12-13 08:13:08,828 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/api.py", line 250, in wrapper
2017-12-13 08:13:08,828 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return old_api(self, *args, **kwargs)
2017-12-13 08:13:08,828 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/addons/report/models/report.py", line 147, in get_html
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return self.render(cr, uid, [], report.report_name, docargs, context=context)
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/api.py", line 250, in wrapper
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return old_api(self, *args, **kwargs)
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/addons/report/models/report.py", line 123, in render
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return view_obj.render(cr, uid, template, values, context=context)
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/api.py", line 250, in wrapper
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return old_api(self, *args, **kwargs)
2017-12-13 08:13:08,829 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/addons/web_editor/models/ir_ui_view.py", line 29, in render
2017-12-13 08:13:08,830 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return super(view, self).render(cr, uid, id_or_xml_id, values=values, engine=engine, context=context)
2017-12-13 08:13:08,830 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/api.py", line 250, in wrapper
2017-12-13 08:13:08,830 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return old_api(self, *args, **kwargs)
2017-12-13 08:13:08,830 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_ui_view.py", line 1087, in render
2017-12-13 08:13:08,830 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return self.pool[engine].render(cr, uid, id_or_xml_id, qcontext, loader=loader, context=context)
2017-12-13 08:13:08,830 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/api.py", line 250, in wrapper
2017-12-13 08:13:08,830 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return old_api(self, *args, **kwargs)
2017-12-13 08:13:08,831 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 254, in render
2017-12-13 08:13:08,831 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return self.render_node(element, qwebcontext, generated_attributes=qwebcontext.pop('generated_attributes', ''))
2017-12-13 08:13:08,831 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 297, in render_node
2017-12-13 08:13:08,831 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self.render_element(element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,831 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,831 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,831 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 295, in render_node
2017-12-13 08:13:08,832 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self._render_tag[t_render](self, element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,832 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 448, in render_tag_call
2017-12-13 08:13:08,832 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     d[0] = self.render_element(element, template_attributes, generated_attributes, d)
2017-12-13 08:13:08,832 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,832 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,832 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 295, in render_node
2017-12-13 08:13:08,832 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self._render_tag[t_render](self, element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,833 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 426, in render_tag_foreach
2017-12-13 08:13:08,833 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     ru.append(self.render_element(element, template_attributes, generated_attributes, copy_qwebcontext))
2017-12-13 08:13:08,833 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,833 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,834 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 295, in render_node
2017-12-13 08:13:08,834 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self._render_tag[t_render](self, element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,834 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 459, in render_tag_call
2017-12-13 08:13:08,834 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     res = self.render(cr, uid, template, d)
2017-12-13 08:13:08,834 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/api.py", line 250, in wrapper
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return old_api(self, *args, **kwargs)
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 254, in render
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     return self.render_node(element, qwebcontext, generated_attributes=qwebcontext.pop('generated_attributes', ''))
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 297, in render_node
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self.render_element(element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 295, in render_node
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self._render_tag[t_render](self, element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,835 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 448, in render_tag_call
2017-12-13 08:13:08,836 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     d[0] = self.render_element(element, template_attributes, generated_attributes, d)
2017-12-13 08:13:08,836 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,836 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,836 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 297, in render_node
2017-12-13 08:13:08,836 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self.render_element(element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,836 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,836 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 297, in render_node
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self.render_element(element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 297, in render_node
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self.render_element(element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,837 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,838 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 295, in render_node
2017-12-13 08:13:08,838 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self._render_tag[t_render](self, element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,838 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 426, in render_tag_foreach
2017-12-13 08:13:08,838 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     ru.append(self.render_element(element, template_attributes, generated_attributes, copy_qwebcontext))
2017-12-13 08:13:08,838 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,838 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,838 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 297, in render_node
2017-12-13 08:13:08,839 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self.render_element(element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,839 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 325, in render_element
2017-12-13 08:13:08,839 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     raise_qweb_exception(message="Could not render element %r" % element.tag, node=element, template=template)
2017-12-13 08:13:08,839 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 320, in render_element
2017-12-13 08:13:08,839 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     generated_attributes= name == "t" and generated_attributes or ''))
2017-12-13 08:13:08,839 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 295, in render_node
2017-12-13 08:13:08,839 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     result = self._render_tag[t_render](self, element, template_attributes, generated_attributes, qwebcontext)
2017-12-13 08:13:08,840 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `   File "/home/travis/odoo-9.0/openerp/addons/base/ir/ir_qweb.py", line 503, in render_tag_field
2017-12-13 08:13:08,840 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: `     field = record._fields[field_name]
2017-12-13 08:13:08,840 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: ` QWebException: 'NoneType' object has no attribute '_fields'
2017-12-13 08:13:08,840 5240 INFO openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: Ran 1 test in 5.444s
2017-12-13 08:13:08,840 5240 ERROR openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking: FAILED
2017-12-13 08:13:08,840 5240 INFO openerp_test openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking:  (errors=1)
2017-12-13 08:13:08,840 5240 INFO openerp_test openerp.modules.module: openerp.addons.account_invoice_group_picking.tests.test_account_invoice_group_picking tested in 5.46s, 4729 queries
2017-12-13 08:13:08,841 5240 ERROR openerp_test openerp.modules.module: Module account_invoice_group_picking: 0 failures, 1 errors
2017-12-13 08:13:09,203 5240 INFO openerp_test openerp.modules.module: module account_invoice_merge_purchase: creating or updating database tables
2017-12-13 08:13:09,292 5240 INFO openerp_test openerp.modules.module: openerp.addons.account_invoice_merge_purchase.tests.test_account_invoice_merge_purchase running tests.
2017-12-13 08:13:09,994 5240 INFO openerp_test openerp.addons.account_invoice_merge_purchase.tests.test_account_invoice_merge_purchase: test_picking_multi_purchase_order (openerp.addons.account_invoice_merge_purchase.tests.test_account_invoice_merge_purchase.TestAccountInvoiceMergePurchase)
2017-12-13 08:13:11,185 5240 INFO openerp_test openerp.models.unlink: User #1 deleted stock.move.operation.link records with IDs: [14]
2017-12-13 08:13:11,222 5240 INFO openerp_test openerp.models.unlink: User #1 deleted stock.move.operation.link records with IDs: [15]
```

**Comment by rafaelbn** [comment]
Migrated to v11 in https://github.com/OCA/account-invoice-reporting/pull/72

**Comment by pedrobaeza** [comment]
Closing as it has been renamed an moved to proper place in v11, and this won't be continued.

**Pull request closed without merging**